### PR TITLE
Read the docs config

### PR DESCRIPTION
**Pull request by the-greenman**
I've now created a default RTD config file, and pinned to an older version of the docutils.  

Doc update docs to remove all sphinx warnings.

### CODE REVIEW

**Comment by lukestanley** [comment]
I see the pinning and I get that, what are the documentation changes about? @the-greenman

**Comment by lukestanley** [comment]
e.g why "Removed complex form example"? @the-greenman

File: docs/workflow/blocks/form.rst [hunk 1]
```diff
@@ -80,64 +80,4 @@ A simple search form without a submit button.
         }
     },
     "uiSchema": {}
-  }
```
Comment (lukestanley) [inline docs/workflow/blocks/form.rst:83]: I didn't get the purpose of removing this yet and just wanted to check.

**Review by lukestanley — APPROVED** [review]

**Comment by the-greenman** [comment]
Ah. I actually thought those had already been merged! 
I went through all the docs and removed all the warnings from Sphinx.

File: docs/workflow/blocks/form.rst (hunk 1 above)
Comment (the-greenman) [reply docs/workflow/blocks/form.rst:83]: I'll bring the complex example back. I was just removing all the updates added at the time of the break.

**Pull request merged**